src/access/whereOwnerOrAdmin.ts

```diff
@@ -0,0 +1,21 @@
+import {Access} from 'payload'
+import { ROLE_SUPER_ADMIN} from '@/utilities/constants'
+
+export const whereOwnerOrAdmin = ({ req:{ user }  }) => {
+  if (user?.role === ROLE_SUPER_ADMIN) {
```

Comment:
I wonder if we really want to have a super admin able to touch all the data of other installations. Maybe we should grant the super admin only access to stuff like users but no to the installations and production data?

Reply:
done